indico/modules/events/registration/models/registrations.py

```diff
@@ -289,6 +289,15 @@ def can_be_modified(self):
         regform = self.registration_form
         return regform.is_modification_open and regform.is_modification_allowed(self)
 
+    @property
+    def can_be_withdrawn(self):
+        if self.is_paid:
+            return False
+        elif self.event.end_dt < now_utc():
+            return False
+        else:
+            return True
```

Comment:
```suggestion
        return not self.is_paid and self.event.end_dt > now_utc()
```

I think here we don't lose any readability by using a more compact version